README.md

```diff
@@ -7,27 +7,33 @@
 [![Code Style: Blue](https://img.shields.io/badge/code%20style-blue-4495d1.svg)](https://github.com/invenia/BlueStyle)
 [![DOI](https://zenodo.org/badge/170755855.svg)](https://zenodo.org/badge/latestdoi/170755855)
 
-
 JLSO is a storage container for serialized Julia objects.
-Think of it less as a serialization format but as a container,
-that employs a serializer, and a compressor, handles all the other concerns including metadata and saving.
-Such that the serializer just needs to determine how to turn a julia object into a stream`Vector{UInt8}`,
-and the compressor just needs to determine how to turn one stream of `UInt8`s into a smaller one (and the reverse).
+Think of it less as a serialization format and more of a container for arbitrarily serialized objects and metadata.
```

Comment:
The point of calling it a container was not merely that that it contains things.
It is specifically referring to a container format
https://en.wikipedia.org/wiki/Container_format_(computing)
such as the Real Media format https://en.wikipedia.org/wiki/RealMedia
which can in turn wrap multiple different codecs/formats (serializers in our case)

With the key untility of JLSO that the serializer and compressors just have to work with `Vector{UInt8}` and JLSO takes care of hooking them up and of metadata.
(which is very much what container formats do)